src/viur/core/bones/base.py

```diff
@@ -132,12 +133,12 @@ class MultipleConstraints:  # Used to define constraints on multiple bones
     The MultipleConstraints class is used to define constraints on multiple bones, such as the minimum
     and maximum number of entries allowed and whether duplicate values are allowed.
     """
-    minAmount: int = 0  # Lower bound of how many entries can be submitted
+    min: int = 0  # Lower bound of how many entries can be submitted
     """An integer representing the lower bound of how many entries can be submitted (default: 0)."""
-    maxAmount: int = 0  # Upper bound of how many entries can be submitted
+    max: int = 0  # Upper bound of how many entries can be submitted
     """An integer representing the upper bound of how many entries can be submitted (default: 0)."""
-    preventDuplicates: bool = False  # Prevent the same value of being used twice
-    """A boolean value indicating if the same value can be used twice (default: False)."""
+    duplicates: bool = True  # Prevent the same value of being used twice
```

Comment:
Since you negated the flag, this should be flipped as well
```suggestion
    duplicates: bool = True  # Allows the same value of being used twice
```

---
Btw; IMO it's totally nonsense to put a comment behind it and having an docstring. Can we extend the docstring and remove these inline comments. Then they would appear in our docs too.